src/analysis/normed_space/multilinear.lean

```diff
@@ -730,19 +743,78 @@ protected def pi_field_equiv : G ≃L[𝕜] (continuous_multilinear_map 𝕜 (λ
 
 end continuous_multilinear_map
 
-open continuous_multilinear_map
-
 lemma continuous_linear_map.norm_comp_continuous_multilinear_map_le
   (g : G →L[𝕜] G') (f : continuous_multilinear_map 𝕜 E G) :
   ∥g.comp_continuous_multilinear_map f∥ ≤ ∥g∥ * ∥f∥ :=
+continuous_multilinear_map.op_norm_le_bound _ (mul_nonneg (norm_nonneg _) (norm_nonneg _)) $ λ m,
+calc ∥g (f m)∥ ≤ ∥g∥ * (∥f∥ * ∏ i, ∥m i∥) : g.le_op_norm_of_le $ f.le_op_norm _
+           ... = _                        : (mul_assoc _ _ _).symm
+
+namespace multilinear_map
+
+/-- Given a map `f : G →ₗ[𝕜] multilinear_map 𝕜 E G'` and an estimate
+`H : ∀ x m, ∥f x m∥ ≤ C * ∥x∥ * ∏ i, ∥m i∥`, construct a continuous linear
+map from `G` to `continuous_multilinear_map 𝕜 E G'`.
+
+In order to lift, e.g., a map `f : (multilinear_map 𝕜 E G) →ₗ[𝕜] multilinear_map 𝕜 E₁ G'`
+to a map `(continuous_multilinear_map 𝕜 E G) →L[𝕜] continuous_multilinear_map 𝕜 E₁ G'`,
+one can apply this construction to `f.comp continuous_multilinear_map.to_multilinear_map_linear`. -/
```

Comment:
```suggestion
one can apply this construction to `f.comp continuous_multilinear_map.to_multilinear_map_linear`. -/
```
Can you add at the end of the sentence something like "which is a linear map from `continuous_multilinear_map 𝕜 E G` to `multilinear_map 𝕜 E₁ G'`", to help the reader parse what is going on?